app/assets/javascripts/views/project_view.js

```diff
@@ -10,26 +11,26 @@ module.exports = Backbone.View.extend({
   initialize: function() {
     _.bindAll(this, 'addStory', 'addAll', 'render');
 
+    this.$loadingSpin = $('.loading-spin');
+
     this.model.stories.on('add', this.addStory);
     this.model.stories.on('reset', this.addAll);
     this.model.stories.on('all', this.render);
     this.model.on('change:userVelocity', this.addAll);
     this.listenTo(this.model, 'change:current_flow', this.prepareColumns);
 
-    this.prepareColumns();
 
-    var that = this;
-    this.model.stories.fetch({
-      success: function() {
-        that.addAll();
-      }
-    });
+    this.prepareColumns();
+    this.$loadingSpin.show();
+    this.model.stories.fetch({success: this.addAll});
   },
 
   prepareColumns: function() {
+    var $columnToggles = this.$el.parent().find('#column-toggles');
```

Comment:
We should not access the parent of the view from inside of it, it breaks isolation, can we avoid it somehow?